demo/index.html

```diff
@@ -132,7 +132,9 @@ <h1>Symphony Electron API Demo</h1>
 
             num++;
 
-            var notf = new ssf.Notification(title, {
```

Comment:
Can we retain this till we completely move to context isolation?

Reply:
No. In this branch, `ssf` does not exist